content/docs/capabilities/tcp/client.mdx

```diff
@@ -22,21 +22,32 @@ import LongLivedConnections from '@site/content/docs/admonitions/_long-lived-con
 
 # Pomerium Desktop and CLI Clients
 
-Pomerium is capable of creating secure connections to services like SSH, Redis, and more by creating a TCP tunnel to the service with a local client. This article describes configuring a route to accept TCP connections, and using either the CLI or GUI client to connect to it.
+This document describes how to use Pomerium's Desktop and CLI clients to connect to TCP routes in Pomerium.
 
-## Create a TCP Route
+:::note TCP Terminology
 
-1. Specify this new Route as a TCP Route by prefixing `tcp+` in the **From** field, along with a port suffix.
+In this document, a "TCP route" refers to a route configured to accept TCP connections.
```

Comment:
I think this is a good idea — it's natural for a reader to have the question "what's a TCP route" here.

I think we could make this even clearer by comparing with "regular" http/https routes.

What do you think of something like this?

```markdown
:::info What's a TCP route?

A TCP route is for use with upstream services that don't speak HTTP — for example
SSH, Redis, or MySQL. With a TCP route, the entire connection is proxied to the
upstream service, rather than each individual request separately.

In Pomerium, TCP routes are denoted with a `tcp+` prefix in the route's **From** URL.
```